### ISSUE TITLE
Image does not work on Pi 4

### ISSUE THREAD

**Issue by MathewH** [iizukanao/picam-streamer#41]
Hi,

I've tried to get the latest image to boot on a Raspberry Pi 4, but have been unsuccessful.
I have tried booting the image on a Pi 3 and updating + upgrading without success.
The Pi runs fine with a Raspbian Buster light image, which I tried to exclude failures on the hardware side.
Would it be possible to get a new image or instructions on how to install & configure Picam-Streamer on an existing system?

Kind Regards,
Mathias

**Comment by MathewH** [comment]
Update: I have tried upgrading the image to buster by editing /etc/apt/sources.list and /etc/apt/sources.list.d/raspi.list to the new sources. While the upgrade worked fine, it still won't boot on a Raspberry Pi 4.

**Comment by MathewH** [comment]
Update2: From what I have read, the upgrade to Buster should work with the new Pi 4.
After confirming with  'cat /etc/*-release' I got:
PRETTY_NAME="Raspbian GNU/Linux 10 (buster)"
NAME="Raspbian GNU/Linux"
VERSION_ID="10"
VERSION="10 (buster)"
VERSION_CODENAME=buster
ID=raspbian
ID_LIKE=debian
HOME_URL="http://www.raspbian.org/"
SUPPORT_URL="http://www.raspbian.org/RaspbianForums"
BUG_REPORT_URL="http://www.raspbian.org/RaspbianBugs"

So the update should have been completed fine. I will continue searching and come back once I have found something.

**Comment by MathewH** [comment]
Update 3: So, after comparing the different images, it seems that upgrading to Buster leaves the boot partition mostly unchanged, there wouldn't even be enough size. Maybe there is a way to just upgrade the OS and the boot segment while leaving the programs intact, but it seems like starting fresh from a new Buster image seems to be easier.

**Comment by MathewH** [comment]
Final Update: After testing multiple options, it seems updating the old image to buster is not feasible for me. So instead I opted for installing picam + node-rtsp-rtmp-server on a new image.
I used https://hmbd.wordpress.com/2016/08/01/raspberry-pi-video-and-audio-recording-and-streaming-guide/ as a guideline.
This lacks the Strobe Media Playback for browsers, but was enough for my use case.
I am going to close this issue, since my problem was resolved and maybe someone who has a similar problem might find this helpful.
